### DISCUSSION TITLE
🔗 Buttons: Separate Typography Options for Normal and Hover States (Merged with #30211)

### DISCUSSION THREAD

**Discussion by enuzzo** [elementor/elementor#21945] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

If in "Site Settings > Typography > Links" I decide that every link in hover should have "underline" decoration, the same setting applies to the hover in "Site Settings > Button > Hover". I understand the reasoning, as button links are also links, but the two styling cases are quite different. It's possible to want the underline for all global site links but wish for the button links to be set as "text-decoration: none;", because in most cases, the color change is enough to differentiate them.

### Describe the solution you'd like

Allow specifying typography options for "normal" and "hover" states separately in "Site Settings > Button".

### Describe alternatives you've considered

The alternative is to write a CSS rule in the Hello Child's style that overrides the text-decoration for links when they are inside buttons. This is laborious and, more importantly, unpredictable to maintain in terms of design consistency.

**Comment by nicholaszein** [comment]
Hi there,

### 🔃 This Feature Request is being merged with #30211 which is on the same subject. 

if you still think this request should be considered for our roadmap, please cast your **⬆️ Upvote** on the first comment.

Don't worry, your comments, reactions, and participation in this issue will be accounted for when considering this Feature Request for our Roadmap.

Happy voting! 🙂
